### PR TITLE
Update to 1.19.4 (Fixed)

### PR DESCRIPTION
**Pull request by CrytoPal**
## Description

Brings Orion to 1.19.4
## Motivation and Context

Why not
## How Has This Been Tested?

I've built it and tested it with the latest meteor build.
## Screenshots (if appropriate):
Will show if needed
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ x ] Have you successfully ran tests with your changes locally?
Updated stuff to 1.19.4

### CODE REVIEW

**Comment by C10udburst** [comment]
add files via upload

File: src/main/java/me/ghosttypes/orion/modules/main/AnchorAura.java [hunk 1]
```diff
@@ -257,12 +257,12 @@ private BlockPos findBreakPos(BlockPos targetPos) {
         return null;
     }
 
-    private boolean getDamagePlace(BlockPos pos) {
-        return placeMode.get() == Safety.Suicide || DamageUtils.bedDamage(mc.player, Utils.vec3d(pos.add(0.5, 0.5, 0.5))) <= maxDamage.get();
+    private boolean getDamagePlace(BlockPos pos) { // Thanks Crosby :)
```
Comment (C10udburst) [inline src/main/java/me/ghosttypes/orion/modules/main/AnchorAura.java:260]: :raised_eyebrow: elaborate please

Comment (CrytoPal) [reply src/main/java/me/ghosttypes/orion/modules/main/AnchorAura.java:260]: Crosby told me how to fix the issue on discord 💀

Comment (tympanicblock61) [reply src/main/java/me/ghosttypes/orion/modules/main/AnchorAura.java:260]: Bro did not, he told me

Comment (tympanicblock61) [reply src/main/java/me/ghosttypes/orion/modules/main/AnchorAura.java:260]: Screenshot_20230416-195744

Comment (RacoonDog) [reply src/main/java/me/ghosttypes/orion/modules/main/AnchorAura.java:260]: > Bro did not, he told me

shut up i told both of yall

**Pull request merged**